contracts/context-config/src/query.rs

```diff
@@ -28,6 +28,15 @@ impl ContextConfigs {
         context.application.revision()
     }
 
+    pub fn proxy_contract(&self, context_id: Repr<ContextId>) -> AccountId {
+        let context = self
+            .contexts
+            .get(&context_id)
+            .expect("context does not exist");
+
+        context.proxy.clone()
```

Comment:
reference should suffice

```suggestion
    pub fn proxy_contract(&self, context_id: Repr<ContextId>) -> &AccountId {
        let context = self
            .contexts
            .get(&context_id)
            .expect("context does not exist");

        &context.proxy
```